SimpleWAWS.Console/Program.cs

```diff
@@ -3,6 +3,7 @@
 using System;
```

Comment:
When we do the switch over we'll need to delete and recreate the resourcegroups with the functionapps and storage account in them. I dont know if the code is smart enough  to detect that there are no storage accounts and recreate them